### DISCUSSION TITLE
How to disable the Theme ?

### DISCUSSION THREAD

**Discussion by yellowbean** [reflex-dev/reflex#2982] [Q&A] [answered]
I can use the short-cut in Mac to close this every time ( it is pretty annoying and distraction, it will hide html elements, especially at early stage of development which doesn't pay much attention to the `style` ) 

but I can't not do it in Windows , which key binding to remove this ? 

Is this related to `tailwind`?  I've set this field to `None` but it still shows up 

```
config = rx.Config(
    app_name="prophet",
    frontend_port=3001,
    loglevel="debug",
    tailwind=None
)
```

![image](https://github.com/reflex-dev/reflex/assets/1008321/9284e1a8-afa2-4682-96b1-d7dddd30ed44)

**Comment by picklelo** [comment, marked as answer]
Just remove the `rx.theme_panel()` line from within your UI code and it will go away
